sources/packages/forms/src/form-definitions/reportscholasticstandingchange.json

```diff
@@ -1445,399 +1379,593 @@
+                                        "height": ""
+                                    },
+                                    "allowCalculateOverride": false,
+                                    "encrypted": false,
+                                    "showCharCount": false,
+                                    "showWordCount": false,
+                                    "properties": {},
+                                    "allowMultipleMasks": false,
+                                    "addons": [],
+                                    "tag": "p",
+                                    "id": "ezkptku"
+                                },
+                                {
+                                    "label": "Data Grid",
+                                    "labelPosition": "top",
+                                    "labelWidth": "",
+                                    "labelMargin": "",
+                                    "description": "",
+                                    "tooltip": "",
+                                    "disableAddingRemovingRows": true,
+                                    "conditionalAddButton": "",
+                                    "reorder": false,
+                                    "addAnother": "",
+                                    "addAnotherPosition": "bottom",
+                                    "layoutFixed": false,
+                                    "enableRowGroups": false,
+                                    "initEmpty": false,
+                                    "customClass": "formio-unique-datagrid",
+                                    "tabindex": "",
+                                    "hidden": false,
+                                    "hideLabel": true,
+                                    "autofocus": false,
+                                    "disabled": false,
+                                    "tableView": false,
+                                    "modalEdit": false,
+                                    "defaultValue": [
+                                        {}
+                                    ],
+                                    "persistent": true,
+                                    "protected": false,
+                                    "dbIndex": false,
+                                    "encrypted": false,
+                                    "redrawOn": "",
+                                    "clearOnHide": true,
+                                    "customDefaultValue": "",
+                                    "calculateValue": "const divEl = document.getElementsByClassName('formio-unique-datagrid')[0]\r\nconst tableEl = divEl.getElementsByTagName('table')[0]\r\nconst tabelHeadEl = tableEl.getElementsByTagName('thead')[0]\r\ntableEl.classList.remove(\"table\");\r\ntableEl.classList.remove(\"datagrid-table\");\r\ntableEl.classList.remove(\"table-bordered\");\r\ntabelHeadEl.setAttribute(\"style\", \"display: none\");",
```

Comment:
Cant it be resolved with a custom CSS? I am assuming that this is to provide the style to the below area, right?
![image](https://user-images.githubusercontent.com/61259237/163861108-d096dc82-20e8-4d6b-babe-baa6f3977c75.png)